source/views/stoprint/components/error.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import React from 'react'
+import type {TopLevelViewPropsType} from '../../types'
+import {View, StyleSheet, Platform} from 'react-native'
+import Icon from 'react-native-vector-icons/Ionicons'
+import {NoticeView} from '../../components/notice'
+import * as c from '../../components/colors'
+
+const ERROR_MESSAGE =
```

Comment:
How awful would it be to just put all of this on one line and remove the backslashes?

Reply:
lol not so awful I guess